### DISCUSSION TITLE
Do metaSPAdes and metaviralSPAdes share same error correction algorithm?

### DISCUSSION THREAD

**Discussion by kodragonP** [ablab/spades#832] [Q&A] [answered]
### Description of bug

Can I use the error corrected reads produced by metaSPAdes to run metaviralSPAdes directly?

### spades.log

[spades.log](https://github.com/ablab/spades/files/7063324/spades.log)

### params.txt

[params.txt](https://github.com/ablab/spades/files/7063323/params.txt)

### SPAdes version

SPAdes v 3.15.3

### Operating System

ubantu

### Method of SPAdes installation

conda

### No errors reported in spades.log

- [X] Yes

**Comment by asl** [comment, marked as answer]
Sure
